source/core/parameters_lagrangian.cc

```diff
@@ -65,15 +65,17 @@ namespace Parameters
       const unsigned int &particle_type,
       ParameterHandler   &prm)
     {
-      const std::string size_distribution_type =
-        prm.get("size distribution type");
-      if (size_distribution_type == "uniform")
+      const std::string size_distribution = prm.get("size distribution type");
```

Comment:
I would use "distribution function" or "distribution function type" for the parameter instead of using size again here. And for the variable, distribution_function or distribution_type would make more sense imo

Reply:
Are you talking about the string in the prm.get ?

Reply:
Both

Reply:
Ok, I'm not sure if I agree. I think it's fine the way it is right not. 

1. Changing the get string would break compatibility with all the prm file. It's not a problem for us, put for all the users out there it's better not to (except when we don't have the choice).  
2. For a random user, when you say "distribution function" it's not clear what the distribution is about. Like you could be talking about a distribution for the rolling resistance coefficient or something. I think keeping the word "size" or "diameter" is better. 

It could do this if you prefer: `const std::string size_distribution_type_str = prm.get("size_distribution_type");`

Reply:
Although we never use _ in parameter names
we use spaces

Reply:
My mistake, you are right. It's a typo

const std::string size_distribution_type_str = prm.get("size distribution type")


I have added a "_str", that is what I was talking about.

Reply:
I got it. Nope, using the type in the variable name is not necessary. You are right to keep it as it is